CONTRIBUTING.md

```diff
@@ -70,20 +77,40 @@ We have a sophisticated CI process setup which gets run on every PR. You must pa
 Once your changes are ready, you can raise a PR. The title of your PR should match the following format:
 
 ```text
-<tag>(<package>): <short description>
+<type>(<package>): <Subject> (#Issue Number)
```

Comment:
The `(#Issue Number)` gets added by GitHub. We don't need it.

Also, I kind of like _short description_ more than `Subject` TBH. It's more clear what it's supposed to be: short, not a whole long ordeal. Thoughts?